extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts

```diff
@@ -648,21 +640,33 @@ export class MigrationCutoverDialog {
 
 				this._fileCount.value = loc.ACTIVE_BACKUP_FILES_ITEMS(tableData.length);
 
-				// Sorting files in descending order of backupStartTime
-				tableData.sort((file1, file2) => new Date(file1.backupStartTime) > new Date(file2.backupStartTime) ? - 1 : 1);
-
-				this._fileTable.data = tableData.map((row) => {
-					return [
-						row.fileName,
-						row.type,
-						row.status,
-						row.dataUploaded,
-						row.copyThroughput,
-						convertIsoTimeToLocalTime(row.backupStartTime).toLocaleString(),
-						row.firstLSN,
-						row.lastLSN
-					];
-				});
+				if (tableData.length === 0) {
+					this._emptyTableFill.updateCssStyles({
+						'display': 'flex'
+					});
+					this._fileTable.height = '50px';
+				} else {
+					this._emptyTableFill.updateCssStyles({
+						'display': 'none'
+					});
+					this._fileTable.height = '300px';
+
+					// Sorting files in descending order of backupStartTime
+					tableData.sort((file1, file2) => new Date(file1.backupStartTime) > new Date(file2.backupStartTime) ? - 1 : 1);
+
+					this._fileTable.data = tableData.map((row) => {
+						return [
+							row.fileName,
+							row.type,
+							row.status,
+							row.dataUploaded,
+							row.copyThroughput,
+							convertIsoTimeToLocalTime(row.backupStartTime).toLocaleString(),
```

Comment:
please verify this handles an undefined backupStartTime

Reply:
backupStartTime comes from BackupSetInfo.backupStartDate, which a required string object. If it is an empty string '', convertIsoTimeToLocalTime(row.backupStartTime).toLocaleString() will return "Invalid Date"